packages/esm-providers-app/src/table/provider-data-table.component.tsx

```diff
@@ -26,19 +26,62 @@ import { ConfigObject } from '../config-schema';
 import dayjs from 'dayjs';
 import CustomActionMenu from '../overflow/overflow-component';
 import EmptyProviderState from './empty-state.component';
-import { Report, SearchLocate } from '@carbon/react/icons';
+import { Report } from '@carbon/react/icons';
 
-const ProviderListTable: React.FC<{ filter: (provider: any) => boolean }> = ({ filter }) => {
+const ProviderListTable: React.FC = () => {
   const { t } = useTranslation();
   const layout = useLayoutType();
   const responsiveSize = isDesktop(layout) ? 'sm' : 'lg';
   const { provider, error, isLoading } = useProviders();
   const { licenseNumberUuid, licenseExpiryDateUuid, providerNationalIdUuid } = useConfig<ConfigObject>();
-  const [pageSize, setPageSize] = React.useState(10);
-  const [searchQuery, setSearchQuery] = React.useState('');
+  const [pageSize, setPageSize] = useState(10);
+  const [searchQuery, setSearchQuery] = useState('');
+  const [selectedFilter, setSelectedFilter] = useState('All Providers'); // Default filter to "All Providers"
```

Comment:
```suggestion
  const [selectedFilter, setSelectedFilter] = useState<FilterType>('All Providers'); // Default filter to "All Providers"
```

```tsx
type FilterType  = 'All Providers' | 'Active' 
```
we should provide the type of filters allowed here